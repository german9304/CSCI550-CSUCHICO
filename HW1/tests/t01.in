20
0 3
1 3
2 3
3 4
4 5
4 6
6 7
6 8
8 9
8 18
17 18
18 19
9 10
9 11
11 12
11 13
13 14
13 15
15 16
