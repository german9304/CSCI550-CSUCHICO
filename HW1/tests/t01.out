8 6 4 3 0 
8 6 4 3 1 
8 6 4 3 2 
8 6 4 3 
8 6 4 
8 6 4 5 
8 6 
8 6 7 
8 9 
8 9 10 
8 9 11 
8 9 11 12 
8 9 11 13 
8 9 11 13 14 
8 9 11 13 15 
8 9 11 13 15 16 
8 18 17 
8 18 
8 18 19 
